python/lsst/afw/math/_chebyshevBoundedField.py

```diff
@@ -72,3 +72,29 @@ def approximate(cls, boundedField,
         y = np.repeat(ySteps, nStepX)
 
         return cls.fit(bbox, x, y, boundedField.evaluate(x, y), ctrl)
+
+    def evaluateArray(self, x, y):
+        """Evaluate a ChebyshevBoundedField at an array of positions.
+
+        This is a simple shim around evaluate() to use the same API
+        as the Python vectorized lsst.meas.algorithms.CoaddBoundedField.
+
+        Parameters
+        ----------
+        x : `np.ndarray`
+            Array of x values.
+        y : `np.ndarray`
+            Array of y values.
+
+        Returns
+        -------
+        values : `np.ndarray`
+            Array of evaluated values.
+        """
```

Comment:
This should go on the base `BoundedField` class instead, I think.  That will make it work on any bounded field (including chebyshev), while still letting `CoaddBoundedField` override it.